Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -1982,10 +1116,16 @@ void LoadEventNexus::loadEvents(API::Progress *const prog,
 
   for (size_t i = bank0; i < bankn; i++) {
     // We make tasks for loading
-    if (bankNumEvents[i] > 0)
-      pool.schedule(new LoadBankFromDiskTask(
+    if (bankNumEvents[i] > 0) {
+      LoadBankFromDiskTask *task = new LoadBankFromDiskTask(
           this, bankNames[i], classType, bankNumEvents[i], oldNeXusFileNames,
-          prog2, diskIOMutex, scheduler, periodLogVec));
+          prog2, diskIOMutex, scheduler, periodLogVec, this->getLogger());
+      task->run();
+
+      // pool.schedule(new LoadBankFromDiskTask(
```

Comment:
We shouldn't keep commented out code in the code base as it's confusing for future developers.